admin/app/views/spree/admin/integrations/_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if @integration.form_partial_name.present? %>
```

Comment:
@mad-eel let's simplify this to only `form_partial_name` please, also we can just do 

```
render @integration.to_partial_path
```